node/src/accountant/mod.rs

```diff
@@ -1572,24 +1582,18 @@ mod tests {
             .start()
             .recipient();
         let mut subject = AccountantBuilder::default().build();
-        let unadjusted_account_1 = QualifiedPayableAccount::new(
-            make_payable_account(111_111),
-            1234567,
-            CreditorThresholds::new(1111111),
-        );
-        let unadjusted_account_2 = QualifiedPayableAccount::new(
-            make_payable_account(999_999),
-            444555666,
-            CreditorThresholds::new(111111111),
-        );
-        let adjusted_account_1 = PayableAccount {
-            balance_wei: gwei_to_wei(55_550_u64),
-            ..unadjusted_account_1.bare_account.clone()
-        };
-        let adjusted_account_2 = PayableAccount {
-            balance_wei: gwei_to_wei(100_000_u64),
-            ..unadjusted_account_2.bare_account.clone()
+        let prepare_unadjusted_and_adjusted_payable = |n: u64| {
+            let unadjusted_account = make_meaningless_qualified_payable(n);
+            let adjusted_account = PayableAccount {
+                balance_wei: gwei_to_wei(n / 3),
```

Comment:
The number 3 should be outside and you may prevent 0s by using n + 1 instead. Also, if you could utilise this function elsewhere maybe make it a test utility fn.